packages/web-components/src/components/search-with-typeahead/search-with-typeahead.ts

```diff
@@ -39,6 +39,7 @@ const gridBreakpoint = parseFloat(breakpoints.lg.width) * baseFontSize;
  * @csspart open-button - The button to show the search box. Usage `c4d-search-with-typeahead::part(open-button)`
  * @csspart close-button - The button to hide the search box. Usage `c4d-search-with-typeahead::part(close-button)`
  * @csspart search-input - The input box for search. Usage `c4d-search-with-typeahead::part(search-input)`
+ * @csspart header-search-actions - the container for the search bar. Usage `c4d-search-with-typeahead::part(header-search-actions)`
```

Comment:
Lets follow the established convention and capitalize the first word in the description:
```suggestion
 * @csspart header-search-actions - The container for the search bar. Usage `c4d-search-with-typeahead::part(header-search-actions)`
```